bigtop_toolchain/manifests/installer.pp

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 
 class bigtop_toolchain::installer {
+  include bigtop_toolchain::jdk11
   include bigtop_toolchain::jdk
```

Comment:
should be renamed to jdk8?

Reply:
At this this time Java 11 is just an option and `bigtop-deploy` also has the puppet file named `jdk.pp`, so I avoid the risk for changing the name.
To keep the code clean `jdk8` might be fine, so if we will find the way to add an option for Java 11 on deployment, that would be the time to chage the file name.